__tests__/endpoints.test.js

```diff
@@ -183,7 +183,50 @@ describe("/api/articles/:article_id/comments", () => {
         .expect(200)
         .then(({body}) => {
             expect(body.comments).toEqual([])
-        })
-    })
+        });
+    });
+
+});
 
-})
\ No newline at end of file
+describe("POST /api/articles/:article_id/comments", () => {
+    test("201: when posting comment it responds with the posted comment", () => {
+        const newComment = {username: 'rogersop',
+                            body: "this is a comment"}
+        return request(app)
+        .post("/api/articles/2/comments")
+        .send(newComment)
+        .expect(201)
+        .then(({body}) => {
+            expect(body.comment).toEqual({
+                comment_id: 19,
+                article_id: 2,
+                body: "this is a comment",
+                created_at: expect.any(String),
+                votes: 0,
+                author: "rogersop",
+            });
+        });
+    });
+    test("404: when posting comment on an article that doesnt exist", () => {
+        const newComment = {username: 'rogersop',
+                            body: "this is a comment"}
+        return request(app)
+        .post("/api/articles/34/comments")
+        .send(newComment)
+        .expect(404)
+        .then(({body}) => {
+            expect(body.msg).toBe("Not Found");
+        });
+    });
+    test("404: error when posting comment from user that doesn't exist", () => {
+        const newComment = {username: 'beckbeck',
+                            body: "this is a comment"}
+        return request(app)
+        .post("/api/articles/2/comments")
+        .send(newComment)
+        .expect(404)
+        .then(({body}) => {
+            expect(body.msg).toBe("User Doesn't Exist");
+        });
+    });
+});
```

Comment:
Really good tests so far, but there are a few more that should be included for this endpoint.

What should happen when a user makes a request to an invalid article id?
What should happen when a user makes a request with the request body being incomplete/missing properties?
What should happen when a user provides extra properties to the request body?